Source/bmalloc/bmalloc/GigacageKind.h

```diff
@@ -27,8 +27,8 @@
 
 namespace Gigacage {
 
-enum Kind {
-    Primitive,
+enum class Kind {
+    Primitive = 0,
     NumberOfKinds
 };
```

Comment:
I think you can just do `using enum Kind` then you don't need to add the `Kind::` everywhere and it's already namespaced by `Gigacage` anyway.

Reply:
The issue with that is that functions which take a `Gigacage::Kind` would previously also accept an `int`. In particular I was bit by `bmalloc::heapKind`, but there are other functions as well. Do you think it's overkill?